common/flatpak-context.c

```diff
@@ -701,6 +701,10 @@ unparse_filesystem_flags (const char           *path,
     case FLATPAK_FILESYSTEM_MODE_READ_WRITE:
       break;
 
+    case FLATPAK_FILESYSTEM_MODE_NONE:
+      g_string_insert_c (s, 0, '!');
+      break;
```

Comment:
This is somewhat weird for the `--filesystem=` case which also uses `unparse_filesystem_flags`, but uses `--nofilesystem` instead. However, that explictily checks for the none-mode, so its gonna work.